kolibri/core/auth/models.py

```diff
@@ -344,6 +344,21 @@ def infer_dataset(self, *args, **kwargs):
         )
 
 
+def validate_username(value):
+    try:
+        validators.EmailValidator()(value)
```

Comment:
Can just use Django's global `validate_email` method from the same module.

https://github.com/django/django/blob/stable/1.11.x/django/core/validators.py#L240

Reply:
I thought of that, but as Django docs don't mention it I prefered the other method. It's exactly the same. However after creating the validate methods for the other attrs I've done for consistency